### PR TITLE
Add a pipeline for publish release

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

Add a pipeline for publish release the Trino cascading connector

### Why are the changes needed?

Fix: #3

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

Manually test

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -359,6 +359,7 @@ except in compliance with the Server Side Public License, v 1.
                     </execution>
                 </executions>
             </plugin>
+
```
Comment (yuqi1129) [inline pom.xml:362]: Is this change unrelated?

File: maven/assemble-zip.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
     <id>package</id>
     <formats>
-        <format>zip</format>
+        <format>tar.gz</format>
```
Comment (yuqi1129) [inline maven/assemble-zip.xml:13]: Does the file format need to be `tar.gz`?

Comment (diqiu50) [reply maven/assemble-zip.xml:13]: Yes. the ZIP format requires the installation of the `zip` and `unzip` packages, as some Linux distribution do not install them by default.

File: pom.xml (hunk 1 above)
Comment (diqiu50) [reply pom.xml:362]: Yes, I remove it.

File: maven/assemble-zip.xml (hunk 2 above)
Comment (yuqi1129) [reply maven/assemble-zip.xml:13]: ok

**Review by yuqi1129 — APPROVED** [review]

**Pull request merged**